src/Microsoft.AspNet.Mvc.Xml/DefaultWrapperProviderFactoryProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.Mvc.Xml
+{
+    public class DefaultWrapperProviderFactoryProvider : IWrapperProviderFactoryProvider
```

Comment:
I sensed that we probably wanted the factory-provider approach the way ValueProviderFactory-ValueProvider does?...its default provider does this differently(from Options?)...wasn't sure if we should follow the same approach...